src/lib.rs

```diff
@@ -74,6 +98,29 @@ impl Neo4j {
         )
     }
 
+    /// Define Neo4j lab plugins to get started with the database.
+    /// Returns new instance.
+    #[must_use]
+    pub fn with_neo4j_labs_plugin(&self, plugins: Vec<Neo4jLabsPlugin>) -> Self {
+        let mut env_vars = self.env_vars.clone();
+
+        let plugin_names = plugins.into_iter()
+            .map(|p| format!("\"{}\"", p.get_value()))
+            .collect::<Vec<String>>()
+            .join(",");
+
+        let plugin_definition = format!("[{}]", plugin_names);
+
+        env_vars.insert("NEO4JLABS_PLUGINS".to_owned(), plugin_definition);
+
+        Self {
+            version: self.version.clone(),
+            user: self.user.clone(),
+            pass: self.pass.clone(),
+            env_vars,
+        }
+    }
```

Comment:
These builder-style/with APIs usually take ownership of `self`, this removes the question of what happens to the value that you had before and doesn't require you to clone all the things.

```suggestion
    pub fn with_neo4j_labs_plugin(mut self, plugins: Vec<Neo4jLabsPlugin>) -> Self {
        let plugin_names = plugins.into_iter()
            .map(|p| format!("\"{}\"", p)) // can be just `p` if we implement `Display` as above
            .collect::<Vec<String>>()
            .join(",");

        let plugin_definition = format!("[{}]", plugin_names);

        self.env_vars.insert("NEO4JLABS_PLUGINS".to_owned(), plugin_definition);

        self
    }
```

Doing this also means we don't have to add `#[must_use]`, as there is no way to continue without using.